cocotbext/axi/axis.py

```diff
@@ -691,16 +724,18 @@ async def _run(self):
             if tready_sample and tvalid_sample:
                 if not frame:
                     if self.byte_size == 8:
-                        frame = AxiStreamFrame(bytearray(), [], [], [], [])
+                        frame = AxiStreamFrame(bytearray(), [], [], [], [], [])
```

Comment:
These caught me out, and may catch out users. the API change may justify a major version uplift

Reply:
I could mitigate this by adjusting the ordering of `tstrb` in the constructor arguments?